src/Carousel.tsx

```diff
@@ -205,7 +208,7 @@ class Carousel extends React.Component<CarouselProps, CarouselInternalState> {
       this.isAnimationAllowed = false;
     }
     this.setState({
-      transform: -(itemWidth * this.state.currentSlide)
+      transform: (this.state.totalItems < this.state.slidesToShow) ? 0 : -(itemWidth * this.state.currentSlide),
```

Comment:
This change is also not needed as currentSlide will already be 0 if there's isnt enough children.